components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/provisioning/impl/DefaultProvisioningHandler.java

```diff
@@ -84,6 +86,7 @@ public class DefaultProvisioningHandler implements ProvisioningHandler {
     private static final String USER_WORKFLOW_ENGAGED_ERROR_CODE = "WFM-10001";
     private static volatile DefaultProvisioningHandler instance;
     private static final String LOCAL_DEFAULT_CLAIM_DIALECT = "http://wso2.org/claims";
+    private static Boolean allowAssociationToExistingUser;
```

Comment:
Why "Boolean" not boolean?

Reply:
In the `isAssociationToExistingUserAllowed` function we are using a null check to identify if this configuration is loaded or not.